server/controllers/OrderController.js

```diff
@@ -76,6 +76,37 @@ class OrderController extends Controller {
       orders,
     });
   }
+
+  static retrieveDayOrders(req, res) {
```

Comment:
The static controller method is concise and readable but you can use some array functions to simplify the code and make it more readable.

Reply:
Thank you for the feedback. I would implement your recommendation on this file and will also endeavour to go through my code to make changes to similar lines of code.